src/main/resources/assets/twilightforest/lang/en_us.lang

```diff
@@ -72,7 +72,7 @@ advancement.twilight_experiment_115_3.desc=eating experiment one-hundred-fifteen
 
 # Snow Forest Progression
 advancement.twilight_progress_yeti=Alpha Fur
-advancement.twilight_progress_yeti.desc=Line your garments with the soft fur from the alpha yeti, keeping you safe from the glacier's cold.
+advancement.twilight_progress_yeti.desc=Line your garments with the soft fur from the Alpha Yeti, keeping you safe from the glacier's cold.
```

Comment:
If this is referring to the biome, Twilight Glacier is the biome name, but if this is just referring to the glacier as an object it's fine as is.

Reply:
That part I was a bit uncertain about, if it was just the biome being referred to as just a glacier, or the biome itself Twilight Glacier. If the latter is the case, I may also tweak the line that mentions the Twilight Swamp as just "Swamp". In any case, I'll fix that one too.